packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -43,12 +43,6 @@ describe('Integration', () => {
       }
     });
 
-    it('NODE_PATH', async () => {
```

Comment:
Should this test be added back now?

Reply:
In that setup we already define a baseUrl in jsconfig so NODE_PATH from .env would be ignored.